spec/repository_spec.rb

```diff
@@ -26,3 +26,17 @@
     expect(EXTRA_FILES).to eq([])
   end
 end
+
+describe "The file \"renamed-styles.json\"" do
+  it "must be valid JSON" do
+    expect(JSON_IS_VALID).to be true
+  end
```

Comment:
Some of the profile times make no sense to me, though, e.g.:

```
dependent/the-journal-of-heart-and-lung-transplantation: must have a style ID
    0.78908 seconds ./spec/styles_spec.rb:120
```

which should be superfast. I assume there some hiccup during some of these examples, like garbage collection?